### PR TITLE
Exclude metaphlan datamanager from tests

### PR DESCRIPTION
**Pull request by mvdbeek**
Probably causes weekly test failures #5061 

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bwlang** [comment]
I can tell what you did... but not why... 
looks reasonable to me

**Comment by mvdbeek** [comment]
.tt_skip contains paths to tools for which we skip testing. I think the metaphlan database is so big it takes down the github test runner, which means we don't get the summary for the weekly testing results. By skipping it I hope we can get that failing chunk to work again.

**Comment by tuncK** [comment]
It is probably not only because of downloading 2.6GB, but also because of some indexing etc. steps that follow it. I dont know what causes a problem & what does not, but humann DM might also cause similar issues.

File: .tt_skip [hunk 1]
```diff
@@ -1,3 +1,4 @@
+data_manager_metaphlan_database_downloader/
```
Comment (bgruening) [inline .tt_skip:1]: Should this not start with "data_managers/"

Comment (mvdbeek) [inline .tt_skip:1]:
```suggestion
data_managers/data_manager_metaphlan_database_downloader/
```

Comment (mvdbeek) [reply .tt_skip]: Thanks!

**Review by bgruening — APPROVED** [review]
I'm not a big fan of that, we should probably have a separate skip list for the weekly tests.
But to get things moving let's get this in.

**Comment by bgruening** [comment]
Thanks @mvdbeek

**Comment by mvdbeek** [comment]
That was the problem, it's finishing properly now : https://github.com/galaxyproject/tools-iuc/actions/runs/3957256233/jobs/6777633519

**Pull request merged**